Rubberduck.Deployment/BuildRegistryScript.ps1

```diff
@@ -95,9 +95,17 @@ try
 				if (Test-Path -Path $regFile -PathType Leaf)
 				{
 					$datetime = Get-Date;
-					& reg.exe import $regFile;
+					if ([Environment]::Is64BitOperatingSystem)
+					{
+						& reg.exe import $regFile /reg:32;
+						& reg.exe import $regFile /reg:64;
+					}
+					else 
+					{
+						& reg.exe import $regFile /reg:32;					
```

Comment:
I would not pass an explicit `/reg:32` flag here. Just remove it and let it be implicit in this case.